test/e2e/catalog_e2e_test.go

```diff
@@ -183,21 +191,20 @@ var _ = Describe("Catalog", func() {
 
 		// Get updated catalogsource
 		fetchedUpdatedCatalog, err := fetchCatalogSourceOnStatus(crc, mainCatalogName, globalNS, catalogSourceRegistryPodSynced)
-		require.NoError(GinkgoT(), err)
+		Expect(err).ShouldNot(HaveOccurred())
 
 		subscription, err = fetchSubscription(crc, testNamespace, subscriptionName, subscriptionStateUpgradePendingChecker)
-		require.NoError(GinkgoT(), err)
-		require.NotNil(GinkgoT(), subscription)
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(subscription).ShouldNot(BeNil())
 
 		// Ensure the timing
 		catalogConnState := fetchedUpdatedCatalog.Status.GRPCConnectionState
 		subUpdatedTime := subscription.Status.LastUpdated
 		timeLapse := subUpdatedTime.Sub(catalogConnState.LastConnectTime.Time).Seconds()
-		require.True(GinkgoT(), timeLapse < 60)
+		Expect(timeLapse < 60).To(BeTrue())
```

Comment:
There are [provided matchers](https://onsi.github.io/gomega/#working-with-numbers-and-times) for `time.Time` and numeric type comparisons that could be used here to get more useful output on failures, if you want to use them.